docs/resim/metrics/metrics_builds.md

```diff
@@ -51,4 +57,25 @@ If the file `/tmp/resim/inputs/batch_metrics_config.json` is present, we are in
 }
```

Comment:
This blob is missing projectID. Also, it can be highlighted by adding beginning with "```json"